wiki/api/v1/endpoints/public.php

```diff
@@ -77,4 +77,24 @@ public function json($path)
             return json_encode($response);
         }
     }
+
+    // Function to return recent edits
+    public function recent($path)
+    {
+        $ActivityQuery = "
+        SELECT `ID`,`PageID`,`AccountID`,`EditTime`,`Size`,`Tags`,`TagList`, `Name`,`Description`,`Title` 
```

Comment:
Instead of directly calling this mysql query, you should create a model for fetching wiki edits.

Reply:
This is how the other API endpoints fetch data -

https://github.com/wetfish/classic/blob/3df2857bade5fea76f0a88e514c70f37590f2a46/wiki/api/v1/endpoints/public.php#L29-L32

Which are defined here - 

https://github.com/wetfish/classic/tree/3df2857bade5fea76f0a88e514c70f37590f2a46/wiki/src/models

You'd likely want to make a new model file called "edits.php"